gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
+    n_obs : integer
+    	number of observations for the obs table
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    astro_table = Table()
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    astro_table.add_column(col_obs_id)
+
+    # on time
+    ontime = Quantity(30.*np.ones_like(col_obs_id.data), 'minute')
+    col_ontime = Column(name='ONTIME', data=ontime)
+    astro_table.add_column(col_ontime)
+
+    # livetime
+    livetime = Quantity(25.*np.ones_like(col_obs_id.data), 'minute')
+    col_livetime = Column(name='LIVETIME', data=livetime)
+    astro_table.add_column(col_livetime)
+
+    # TODO: add columns for coordinates:
+    #       pointing observation -> done
+    #       and date -> done
+    #       then convert to alt az, considering the observatory location
+    # TODO: add column(s) for offset, and take it into account in the coord. transformation!!!
+
+    # RA, Dec
+    # random points on a sphere ref: http://mathworld.wolfram.com/SpherePointPicking.html
+
+    ra = Angle(360.*np.random.random(len(col_obs_id)), 'degree')
+    col_ra = Column(name='RA', data=ra)
+    astro_table.add_column(col_ra)
+
+    dec = Angle(np.arccos(2.*np.random.random(len(col_obs_id)) - 1), 'radian').to('degree')
+    # translate angles from [0, 180) deg to [-90, 90) deg
+    dec = dec - Angle(90., 'degree')
+    col_dec = Column(name='DEC', data=dec)
+    astro_table.add_column(col_dec)
+
+    # date
+    # random points between the start of 2010 and the end of 2014
+    #TODO: should this represent the time at the beginning of the run?
+    #      this has consequences for the ra/dec -> alt/az conversion
+    datestart = Time('2010-01-01T00:00:00', format='fits', scale='utc')
```

Comment:
@mapazarr On travis-ci we currently test with the latest dev and stable version of Astropy. The stable version is currently 1.0.2 (see https://travis-ci.org/gammapy/gammapy/jobs/65301895#L436). So you have to make this example work with Astropy 1.0.2.

In this case you can just use `format=iso` instead of `format=fits`, no?